### PR TITLE
Improve package docs

### PR DESCRIPTION
**Pull request by samueltwum1**
Documentation improvements:
* Package doc available on [ReadTheDocs](https://tw-source-finder.readthedocs.io/en/latest/) with latest build from this branch [here](https://tw-source-finder.readthedocs.io/en/improve-package-docs/).
* README improved to also track doc status

Other:
* Latest changes by @twillis449 pulled from parent repository

### CODE REVIEW

**Comment by samueltwum1** [comment]
@twillis449 Your review of the [docs](https://tw-source-finder.readthedocs.io/en/improve-package-docs/) will be very beneficial in this pull request.

File: README.md [hunk 1]
```diff
@@ -1,40 +1,75 @@
...
+--------
+There are two main scripts in the package, viz: `get_morphology_images` and `get_galaxy_parameters`.
+
+**get_morphology_images**
+
+Uses morphological erosion and dilation to remove background sources from a radio astronomy image. It extends the technique described in [Rudnick, 2002](https://iopscience.iop.org/article/10.1086/342499/pdf).
+
+The process can be described through the following equations:
+
+```python
```
Comment (bngcebetsha) [inline README.md]: if you remove "python" then the markdown will look slightly better than python code

File: README.md [hunk 2]
```diff
@@ -1,40 +1,75 @@
...
+o = original image
+
+d - output from erosion/dilation
```
Comment (bngcebetsha) [inline README.md]: `d =...`

File: README.md [hunk 3]
```diff
@@ -1,40 +1,75 @@
...
+
+The code has been tested with python 3.8 on Ubuntu 20.04. See `pyproject.toml` or `requirements.txt` for package dependencies.
+
+Installation
+------------
+
+Install from source
+
+```bash
+> pip install .
```
Comment (bngcebetsha) [inline README.md]: this and line 74 below...don't we use `$` instead of `>`...this probably not that important

**Review by bngcebetsha — APPROVED** [review]
LGTM

@samueltwum1  I see you noticed a few improvements after using the package. I think we should do more of that, to be discussed in our next catch-up meeting for the release.

**Comment by samueltwum1** [comment]
> LGTM
> 
> @samueltwum1 I see you noticed a few improvements after using the package. I think we should do more of that, to be discussed in our next catch-up meeting for the release.

Yes, I agree. Thanks!

**Pull request merged**